modules/ROOT/pages/api-gateway-encryption-mule4.adoc

```diff
@@ -5,16 +5,16 @@ endif::[]
 
 Gateway startup encryption allows you to encrypt sensitive information stored by the Mule runtime engine such as policies, contracts, and the required credentials to configure the connection between your Mule application and Anypoint Platform.
 
+This feature is supported in Mule versions 4.2.0 and later.
+
 == Encrypting Your Anypoint Platform Credentials Using the Agent
 
 Configuring encryption through the Runtime Manager Agent is the fastest way to secure your client credentials. You have to add the encryption key when creating an Agent server, and the agent will handle the properties encryption for you. +
-See xref:runtime-manager::installing-and-configuring-runtime-manager-agent.adoc[Install and Configure the Runtime Manager Agent] to learn how to configure the agent with encryption.
-
-== Encrypting the Client ID and Secret
+To learn how to configure the agent with encryption, see xref:runtime-manager::installing-and-configuring-runtime-manager-agent.adoc[Install and Configure the Runtime Manager Agent].
 
-If you are not using an agent, you can still manually encrypt your properties.
+== Encrypting the Client ID and Client Secret
 
-The most common way, is using the xref:mule-runtime::secure-configuration-properties.adoc#secure-properties-tool[Secure Properties Tools]. +
+You can manually encrypt your properties even though you are using an agent. The most common way, is using the xref:mule-runtime::secure-configuration-properties.adoc#secure-properties-tool[Secure Properties Tools]. +
 When using encrypted properties, the API gateway assumes that you are using the AES algorithm in CBC Mode.
```

Comment:
```suggestion
When you use encrypted properties, the API gateway assumes that you are using the AES algorithm in CBC Mode.
```
What does it mean that the gateway assumes this? Do you have to use AES in CBC or not?

Reply:
Done! Thanks!